nva-datamodel-java/src/main/resources/publication-ontology.ttl

```diff
@@ -402,7 +402,12 @@ nva:AdditionalIdentifier a rdfs:Class ;
   rdfs:comment "" ;
   rdfs:isDefinedBy nva: ;
   rdfs:subClassOf nva:Identifier .
-  
+
+nva:PublicationNote a rdfs:Class ;
+  rdfs:comment     "" ;
```

Comment:
`    rdfs:comment "An historical string containing information pertaining to the Cristin verification process"`

and likely `    owl:deprecated "true"^^xsd:boolean`

Reply:
Had to set `rdf:type owl:DeprecatedClass ;` as well as you suggestion in order to make it compile. I must admit I have little experience with ontology, can you double check that it is okay?